[IE Python Sample] Update docs (#9807)
openvinotoolkit/openvino: samples/python/ngraph_function_creation_sample/README.md

# nGraph Function Creation Python* Sample {#openvino_inference_engine_ie_bridges_python_sample_ngraph_function_creation_sample_README}

> This sample demonstrates how to execute an inference using [nGraph function feature](../../../docs/nGraph_DG/build_function.md) to create a model that uses weights from LeNet classification model, which is known to work well on digit classification tasks. So you don't need an XML file, the model will be created from the source code on the fly.  

Reviewer: ```suggestion
This sample demonstrates how to run inference using [nGraph function feature](../../../docs/nGraph_DG/build_function.md) to create a model that uses weights from the LeNet classification model, which is known to work well on digit classification tasks. You do not need an XML file, the model is created from the source code on the fly.  
```

| nGraph Functions | [openvino.runtime.op.Parameter], [openvino.runtime.op.Constant], [openvino.runtime.opset8.convolution], [openvino.runtime.opset8.add], [openvino.runtime.opset1.max_pool], [openvino.runtime.opset8.reshape], [openvino.runtime.opset8.matmul], [openvino.runtime.opset8.relu], [openvino.runtime.opset8.softmax] | Description of a model topology using nGraph Python API |

Basic Inference Engine API is covered by [Hello Classification Python* Sample](../hello_classification/README.md).

| Options                    | Values                                                                |
| :------------------------- | :-------------------------------------------------------------------- |
| Validated Models           | LeNet                                                                 |
| Model Format               | Model weights file (\*.bin)                                         |
| Supported devices          | [All](../../../docs/IE_DG/supported_plugins/Supported_Devices.md)     |
| Other language realization | [C++](../../../samples/cpp/ngraph_function_creation_sample/README.md) |

## How It Works

> At startup, the sample application reads command-line parameters, creates a model using [nGraph function feature](../../../docs/nGraph_DG/build_function.md) and passed weights file, loads the model and input data to the Inference Engine plugin, performs synchronous inference, and processes output data, logging each step in a standard output stream.

Reviewer: ```suggestion
At startup, the sample application does the following:
- Reads command line parameters
- Creates a model using [nGraph function feature](../../../docs/nGraph_DG/build_function.md) and passed weights file
- Loads the model and input data to the Inference Engine plugin
- Performs synchronous inference and processes output data, logging each step in a standard output stream

```

You can see the explicit description of
each sample step at [Integration Steps](../../../docs/IE_DG/Integrate_with_customer_application_new_API.md) section of "Integrate the Inference Engine with Your Application" guide.

## Running

> To run the sample, you need specify a model weights and device.

Reviewer: ```suggestion
To run the sample, you need to specify model weights and device.
```

Basic Inference Engine API is covered by [Hello Classification Python* Sample](../hello_classification/README.md).

| Options                    | Values                                                                |
| :------------------------- | :-------------------------------------------------------------------- |
| Validated Models           | LeNet                                                                 |
| Model Format               | Model weights file (\*.bin)                                         |
| Supported devices          | [All](../../../docs/IE_DG/supported_plugins/Supported_Devices.md)     |
| Other language realization | [C++](../../../samples/cpp/ngraph_function_creation_sample/README.md) |

## How It Works

At startup, the sample application does the following:
- Reads command line parameters

> - Creates a model using [nGraph function feature](../../../docs/nGraph_DG/build_function.md) and passed weights file

Reviewer 2: nGraph_DG folder was removed, please change the link to:
```suggestion
- Creates a model using [nGraph function feature](../../../docs/OV_Runtime_UG/model_representation.md) and passed weights file
```
